### PR TITLE
robotic aberrants

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
When using aberrant system on robotic lims it now uses your cog stat rather then bio stat

### CODE REVIEW

**Comment by Guidesu** [comment]
Mechanical limb = MEC stats.

File: code/modules/aberrants/_modification.dm [hunk 1]
```diff
@@ -6,26 +6,29 @@
 	var/install_start_action = "attaching"
 	var/install_success_action = "attached"
 	var/install_time = WORKTIME_FAST
-	//var/install_tool_quality = null				
+	//var/install_tool_quality = null
 	var/install_difficulty = FAILCHANCE_ZERO
 	var/install_stat = STAT_COG
+	var/install_stat_robotic = STAT_COG //Used for robotic organs like FPB bones
```
Comment (benj8560) [inline code/modules/aberrants/_modification.dm:12]:
```suggestion
	var/install_stat_robotic = STAT_MEC //Used for robotic organs like FPB bones
```
I'd be reaaaaallly more happy with this

File: code/modules/aberrants/_modification.dm [hunk 2]
```diff
@@ -6,26 +6,29 @@
...
 	var/install_sound = WORKSOUND_HONK
 
 	var/removal_time = WORKTIME_SLOW
 	var/removal_tool_quality = QUALITY_CLAMPING
 	var/removal_difficulty = FAILCHANCE_CHALLENGING
 	var/removal_stat = STAT_COG
+	var/removal_stat_robotic = STAT_COG //Used for robotic organs like FPB bones
```
Comment (benj8560) [inline code/modules/aberrants/_modification.dm:19]:
```suggestion
	var/removal_stat_robotic = STAT_MEC //Used for robotic organs like FPB bones
```
:D

File: code/modules/aberrants/_modification.dm [hunk 3]
```diff
@@ -6,26 +6,29 @@
...
 
 	var/mod_time = WORKTIME_FAST
-	var/mod_tool_quality = QUALITY_CLAMPING				
+	var/mod_tool_quality = QUALITY_CLAMPING
 	var/mod_difficulty = FAILCHANCE_ZERO
 	var/mod_stat = STAT_COG
+	var/mod_stat_robotic = STAT_COG //Used for robotic organs like FPB bones
```
Comment (benj8560) [inline code/modules/aberrants/_modification.dm:25]:
```suggestion
	var/mod_stat_robotic = STAT_MEC //Used for robotic organs like FPB bones
```

File: code/modules/aberrants/_modification.dm (hunk 1 above)
Comment (benj8560) [reply code/modules/aberrants/_modification.dm:12]: I really just dont understand it what world it makes sense that a scientist would be better suited to this task then robotics?

**Pull request merged**